ddtrace/internal/datadog/profiling/dd_wrapper/include/uploader.hpp

```diff
@@ -24,9 +24,7 @@ class Uploader
   private:
     static inline std::mutex upload_lock{};
     std::string errmsg;
-    static inline std::unique_ptr<ddog_CancellationToken, DdogCancellationTokenDeleter> cancel{
-        ddog_CancellationToken_new()
```

Comment:
If I'm reading this correctly, this was the source of the problem, right? Since there was a non-null cancellation token the first time cancel_inflight was called where there wasn't before?

Reply:
ah, that explains. My brain is not functioning well enough after I was sick last week...  😭